avocado/core/job.py

```diff
@@ -249,7 +249,7 @@ def _make_test_runner(self):
             test_runner_class = runner.TestRunner
 
         self.test_runner = test_runner_class(job=self,
-                                             test_result=self.result_proxy)
+                                             result_proxy=self.result_proxy)
```

Comment:
OK so I take this as a prep work to have one `result` storage and temporarily also the `result_proxy`, right?